website/docs/concepts/blobs.md

```diff
@@ -0,0 +1,42 @@
+---
+id: blobs
+title: Blobs
+sidebar_label: Blobs
+---
+
+import BlobsPng from '@site/static/img/blobs.png';
+
+
+:::info
+
+Only applicable after the Deneb/Cancun Hardfork
+
+:::
+
+<img style={{maxWidth: 760 + 'px'}} src={BlobsPng} />
```

Comment:
![Screenshot_2024-01-31_10-40-11](https://github.com/prysmaticlabs/documentation/assets/7246818/d596b0fb-ade4-425c-85ff-04a3e50c2dae)

Small nitpick: I think this diagram should go after the background so that the reader has some context before being presented with the image.

Reply:
Perhaps we can put "blob retention window ~18 days" under 4844